src/main/java/twentymethods/TwentyMethods.java

```diff
@@ -123,4 +125,83 @@ public static int findIndexOfMatchingIntValueInArrayStartingFromArrayEnd(int[] i
         }
         return -1;
     }
+
+    //10.1) метод принимает инт, и возвращает факториал от заданого инта \
+    public static int getFactorialFromIntFirst(int number) {
+        int result = 1;
+        for (int i = 1; i <= number; i++) {
+            result = result * i;
+        }
+        return result;
+    }
+
+    //10.2) метод принимает инт, и возвращает факториал от заданого инта \
+    public static int getFactorialFromIntSecond(int n) {
+        int result;
+        if (n == 1)
+            return 1;
+        result = getFactorialFromIntSecond(n - 1) * n;
+        return result;
+    }
+
+    //11) принимает инт год, и возвращает тру если год высокосный
+    public static boolean leapCheckYears(int year) {
+        return ((year % 4 == 0) || (year % 100 == 0));
+    }
+
+    //12) приминает массив интов и число, выводит на екран только елементы массива которые кратны этому числу
+    public static void takeMultiplesIntToScreenFirst(int[] ints, int value) {
+        int correctNumber = 0;
+        for (int i : ints) {
+            if (i % value == 0) {
+                correctNumber++;
+                System.out.printf(i + " ");
+            }
+        }
+    }
+
+
+    //13.1) метод принимает массив интов сортирует его по возрастанию
+    public static int[] sortArrayInIncreasingFirst(int[] arr) {
+        Arrays.sort(arr);
+        return arr;
+    }
+
+    //13.2) Sorting метод принимает массив интов сортирует его по возрастанию
+    public static void sortArrayInIncreasingSorting(int[] arrSorting) {
```

Comment:
(int[] arrSorting)
can be just  (int[] arr)

Reply:
done